client/verta/verta/registry/_verta_model_base.py

```diff
@@ -93,6 +93,35 @@ def predict(self, input):
         """
         raise NotImplementedError
 
+    @abc.abstractmethod
```

Comment:
```suggestion
```

`@abc.abstractmethod` makes it required, but this method is actually optional (um I think, right?) so we should omit the decorator.